From 8291c55af779eadbaa6ace720d9c6a40be5b6a04 Mon Sep 17 00:00:00 2001
From: Henry Schreiner <henry.fredrick.schreiner@cern.ch>
Date: Mon, 12 Jan 2026 16:38:46 -0500
Subject: [PATCH 1/3] ci: test on prerelease of 3.15 (#771)

* ci: test on prerelease of 3.15

Signed-off-by: Henry Schreiner <henryfs@princeton.edu>

* fix: handle +

Signed-off-by: Henry Schreiner <henryfs@princeton.edu>

* tests: use / instead of +

Signed-off-by: Henry Schreiner <henryfs@princeton.edu>

---------

Signed-off-by: Henry Schreiner <henryfs@princeton.edu>
---
 .github/workflows/ci.yml | 2 ++
 tests/test_local.py      | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)
 
     steps:
diff --git a/tests/test_local.py b/tests/test_local.py
index e9a0ae5..e31c321 100644
--- a/tests/test_local.py
+++ b/tests/test_local.py
@@ -133,7 +133,7 @@ class TestLocalPath:
             local.path("/opt/lib"),
         ]:
             delta = p.relative_to(src)
-            assert src + delta == p
+            assert src / delta == p
 
     def test_read_write(self):
         with local.tempdir() as dir:
-- 
2.52.0

